### ISSUE TITLE
[Question] User name

### ISSUE THREAD

**Issue by AdrianVovk** [CarbonOS/libsystem#1]
Hello, 

I am creating a Linux distro called carbonOS and I'd like to set up an organization for it. Unfortunately, you have the name. I've noticed that you have no public activity, and was wondering if you'd be willing to change the username on this page.

Sorry for leaving an issue: I couldn't find an email address on your profile.

Thank you for your consideration,
Adrian Vovk

**Comment by AdrianVovk** [comment]
@ovflowd You are the only connection to anything I can find related to this account. Do you own it?

**Comment by ovflowd** [comment]
Hi @AdrianVovk how are you? Sadly no. I even didn't knew before that there was an account here called CarbonOS.

If you need help, you could ask to GitHub support to enter try to solve the issue. But it's likely the wont delete an account in the prol of another one.

The best way it's asking the author of the account. But since there isn't any contact details on his page, it's likely would be impossible, since GitHub would not provide to you his private e-mail.

nvm, good luck :)

**Comment by AdrianVovk** [comment]
@ovflowd I filed with GitHub and they told me that this account is privately active (so they won't give it to me). There is another group (https://GitHub.com/carbon-os) which GitHub will send a message from me to the owners so we can get into contact.

I found you from the followers page on the account. You have the only account I could find (save for maybe analyzing the source code for a username somewhere) related to this project. Thank you for your time, sorry for bothering you.
